adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -186,7 +189,6 @@ class ADAMContext(@transient val sc: SparkContext) extends Serializable with Log
    * @param filePath The (possibly globbed) filepath to load a VCF from.
    * @return Returns a tuple of metadata from the VCF header, including the
    *   sequence dictionary and a list of the samples contained in the VCF.
-   *
```

Comment:
I will refix these. They were fixed at some point but my IDE is not cooperating :(